RecoTracker/MkFitCore/interface/Track.h

```diff
@@ -0,0 +1,683 @@
+      static constexpr int kNSeedHitBits = 4;
+      static constexpr int kMaxSeedHits = (1 << kNSeedHitBits) - 1;
+
+      // Set to true for short, low-pt CMS tracks. They do not generate mc seeds and
+      // do not enter the efficiency denominator.
+      bool not_findable : 1;
+
+      // Set to true when number of holes would exceed an external limit, Config::maxHolesPerCand.
+      // XXXXMT Not used yet, -2 last hit idx is still used! Need to add it to MkFi**r classes.
+      // Problem is that I have to carry bits in/out of the MkFinder, too.
+      bool stopped : 1;
+
+      // Production type (most useful for sim tracks): 0, 1, 2, 3 for unset, signal, in-time PU, oot PU
+      unsigned int prod_type : 2;
+
+      unsigned int align_was_seed_type : 2;
+
+      // Whether or not the track matched to another track and had the lower cand score
+      bool duplicate : 1;
+
+      // Tracking iteration/algorithm
+      unsigned int algorithm : 6;
+
+      // Temporary store number of overlaps for Track here
+      int n_overlaps : 8;
+
+      // Number of seed hits at import time
+      unsigned int n_seed_hits : kNSeedHitBits;
+
+      // mkFit tracking region TrackerInfo::EtaRegion, determined by seed partition function
+      unsigned int eta_region : 3;
+
+      // The remaining bits.
+      unsigned int _free_bits_ : 4;
+
+      Status()
+          : not_findable(false),
+            stopped(false),
+            prod_type(0),
+            align_was_seed_type(0),
+            duplicate(false),
+            algorithm(0),
+            n_overlaps(0),
+            n_seed_hits(0),
+            eta_region(0),
+            _free_bits_(0) {}
+    };
+    static_assert(sizeof(Status) == sizeof(int));
+
+    Status getStatus() const { return status_; }
+    void setStatus(Status s) { status_ = s; }
+
+    bool isFindable() const { return !status_.not_findable; }
+    bool isNotFindable() const { return status_.not_findable; }
+    void setNotFindable() { status_.not_findable = true; }
+
+    void setDuplicateValue(bool d) { status_.duplicate = d; }
+    bool getDuplicateValue() const { return status_.duplicate; }
+    enum class ProdType { NotSet = 0, Signal = 1, InTimePU = 2, OutOfTimePU = 3 };
+    ProdType prodType() const { return ProdType(status_.prod_type); }
+    void setProdType(ProdType ptyp) { status_.prod_type = static_cast<unsigned int>(ptyp); }
+
+    int getNSeedHits() const { return status_.n_seed_hits; }
+    void setNSeedHits(int n) { status_.n_seed_hits = n; }
+    int getEtaRegion() const { return status_.eta_region; }
+    void setEtaRegion(int r) { status_.eta_region = r; }
+
+    // Those are defined in Track, TrackCand has separate member. To be consolidated but
+    // it's a binary format change.
+    // int  nOverlapHits()  const  { return status_.n_overlaps; }
+    // void setNOverlapHits(int n) { status_.n_overlaps = n; }
+
+    /// track algorithm; partial copy from TrackBase.h
+    enum class TrackAlgorithm {
```

Comment:
it's actually a full copy. I've updated the comment in Round 5 follow-up PR trackreco/cmssw#80 commit https://github.com/trackreco/cmssw/pull/80/commits/bb4d80d9c056235c9ee7e1a8344cfbf80f68f7e4

Using directly the CMSSW is currently blocked by using of this enum (as well as the string names) in the standalone.
I added a to-do for a follow-up PR to avoid this dependence [it's not obvious enough to ask to leave it for later; one option is to put this in a light-weight location common for both DataFormats/TrackReco and here in mkFit]